docs/i2c.rst

```diff
@@ -21,6 +21,17 @@ should be described separately in that device's documentation.
 Functions
 =========
 
+.. py:function:: init(freq=None, sda=None, scl=None)
```

Comment:
Updated to use the defaults when no value is specified.

I used "freq" because:

- the `machine` API uses that on other micropython ports
- the mbed internals call it "frequency" internally
- "frequency" seems to be the standard term used anywhere when talking about I2C

Of course changing it to "baudrate" is trivial to do, if needed.